files/KoreBuild/scripts/get-dotnet.sh

```diff
@@ -65,7 +65,7 @@ fi
 channel='preview'
 version=$(__get_dotnet_sdk_version)
 runtime_channel='master'
-runtime_version=$(< "$__script_dir/../config/runtime.version" head -1 | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
+readarray runtime_versions < "$__script_dir/../config/runtime.versions"
```

Comment:
`readarray` requires bash v4 or higher. Just checking in case this will be an issue.

Reply:
The default version of bash on macOS is 3.2 :-/